javascript/javascript2/week3/game.css

```diff
@@ -0,0 +1,47 @@
+body {
+    font-family: 'Times New Roman', Times, serif;
+    background-color: rgb(237, 232, 223)
+}
+
+h1 {
+    padding-left: 30rem;
+
+}
+
+.field {
+    margin-left: 10rem;
+    margin-right: 5rem;
+}
+
+* {
```

Comment:
I would suggest not using the * selector, since it means you are applying the following styles to all children elements indiscriminately.